tests/phpunit/test-wc-stripe-helper.php

```diff
@@ -196,13 +196,13 @@ public function test_get_legacy_individual_payment_method_settings() {
 	}
 
 	/**
-	 * Test for `get_order_by_intent_id`
-	 *
-	 * @param string $status              The order status to return.
-	 * @param bool   $success             Whether the order should be found.
-	 * @return void
-	 * @dataProvider provide_test_get_order_by_intent_id
-	 */
+	* Test for `get_order_by_intent_id`
+	*
+	* @param string $status              The order status to return.
+	* @param bool   $success             Whether the order should be found.
+	* @return void
+	* @dataProvider provide_test_get_order_by_intent_id
+	*/
```

Comment:
```suggestion
	 * Test for `get_order_by_intent_id`
	 *
	 * @param string $status              The order status to return.
	 * @param bool   $success             Whether the order should be found.
	 * @return void
	 * @dataProvider provide_test_get_order_by_intent_id
	 */
```

Not sure what happened here but the indentation is slightly off.